glib-sys/src/lib.rs

```diff
@@ -17,11 +17,11 @@
 
 extern crate libc;
 
-use libc::{c_void, c_int, c_uint, c_float, c_double, c_char, c_uchar, c_long, c_ulong};
+use libc::{c_void, c_int, c_uint, c_float, c_double, c_char, c_uchar, c_long, c_ulong, size_t};
 
 pub type GQuark = u32;
 
-pub type GType = c_ulong;
+pub type GType = size_t;
```

Comment:
This is my current understanding of
```C
#if     GLIB_SIZEOF_SIZE_T != GLIB_SIZEOF_LONG || !defined __cplusplus
typedef gsize                           GType;
#else   /* for historic reasons, C++ links against gulong GTypes */
typedef gulong                          GType;
#endif
```
This change is a noop for unix-like architectures which are LP64 but affects Windows x64 where `sizeof(long) == 4`, so we might want to make extra sure this works there before merging. ;)